tasks/cephfs/fuse_mount.py

```diff
@@ -234,7 +235,12 @@ def umount_wait(self, force=False):
 
         try:
             if self.fuse_daemon:
-                self.fuse_daemon.wait()
+                # Permit a timeout, so that we do not block forever
+                run.wait(self.fuse_daemon, 30)
+        except MaxWhileTries:
+            log.error("process failed to terminate after unmount.  This probably"
+                      "indicates a bug within ceph-fuse.")
+            raise
```

Comment:
@gregsfortytwo in run_tasks.py, in the part following the "Unwinding manager..." log message, it looks to me like exceptions in teardown are explicitly handled.

Entirely possible for a rogue process to be left lying around, but that's why we have nuke